### PR TITLE
Fix path to user home in linux and OSX (#2075, #2076, #2077)

### PR DESCRIPTION
**Pull request by Szczyrk**

### CODE REVIEW

File: Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs [hunk 1]
```diff
@@ -128,14 +131,15 @@ private IEnumerator SetUserDecision(Patcher.UserDecision userDecision)
             Close();
         }
 
-        void OpenFile(string path)
+        void OpenFile(string path, bool isFile)
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs:134]: If this function should open directories as well please rename it to `OpenFileSystemEntry`.

File: Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs [hunk 2]
```diff
@@ -152,19 +156,29 @@ private void OpenLauncherLogFileLocation()
         {
 #if UNITY_EDITOR
             OpenPopup("Access to Launcher in the editor is not possible");
+#else
+#if UNITY_STANDALONE_OSX
+            string logDirectoryPath = Patcher.Instance.Data.Value.LockFilePath.Replace(
+                Patcher.Instance.AppSecret + Path.DirectorySeparatorChar + ".lock", "");
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs:162]: Does it work?

As I remember paths on OSX have shortened, not full, app secret (abcdefgh123456 => abc456)

**Review by witcher112 — CHANGES_REQUESTED** [review]

File: Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs (hunk 1 above)
Comment (Szczyrk) [reply Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs]: Done

File: Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs (hunk 2 above)
Comment (Szczyrk) [reply Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs]: you're right, Hash is the first eight characters. Done

**Review by witcher112 — APPROVED** [review]

**Pull request merged**